src/Microsoft.Health.Fhir.Core/Features/Conformance/Models/ReferenceComponent.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Models
+{
+    public class ReferenceComponent
```

Comment:
Yep, we should wait and use the model in https://github.com/microsoft/fhir-server/pull/792

Reply:
Done